tests/devices/test_quantinuum_device_support.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import pytest
+
+from mqt.bench.devices import QuantinuumProvider
+
+
+def test_quantinuum_provider_methods() -> None:
+    # Test get_available_device_names method
```

Comment:
Instead of having those comments above the four assertions, I would prefer to just have a single docstring below the method definition - although I am aware that we do not yet follow this standard in the other test files.